src/current/_includes/releases/v24.1/v24.1.0-alpha.4.md

```diff
@@ -0,0 +1,116 @@
+- Added a new counter metric, `sql.stats.flush.fingerprint.count`, that tracks the number of unique statement and transaction fingerprints included in the SQL stats flush. [#120522][#120522]
+- Added `node_id`, `dir`, and `wal_failover_path` fields to the `/_status/stores` endpoint. These indicate the store's node ID, data directory, and path to the configured WAL failover secondary, if configured. [#120677][#120677]
+
+<h3 id="v24-1-0-alpha-4-command-line-changes">Command-line changes</h3>
+
+- Added a new `--go-gc-percent` flag for the [`cockroach start`](../v24.1/cockroach-start.html) command that controls the garbage collection target percentage on the Go runtime, mirroring the existing GOGC environment variable. A garbage collection is triggered when the ratio of freshly allocated data to live data remaining after the previous collection reaches this percentage. If left unspecified and a Go soft memory limit is configured (i.e., not explicitly disabled via `--max-go-memory` or `GOMEMLIMIT`), the GC target percentage defaults to 300%. If set to a negative value, the flag disables the target percentage garbage collection heuristic, leaving only the soft memory limit heuristic to trigger garbage collection. [#119605][#119605]
+
+<h3 id="v24-1-0-alpha-4-db-console-changes">DB Console changes</h3>
+
+- The [**Queues** dashboard](../v24.1/ui-queues-dashboard.html) will now display lease queue metrics, similar to the other replica queues. [#119386][#119386]
+- In the DB Console's **SQL Activity** page, the [**Statement Fingerprint**](../v24.1/ui-statements-page.html#statement-fingerprint-page) page now displays "Failure Count" in place of "Failed?", showing the number of failed executions for the given statement fingerprint instead of a "Yes"/"No" value. In addition, previously in the **SQL Activity** table the same statement fingeprint would appear twice: once for failed executions, and once for successful executions. These are no longer considered to be separate rows. Instead, they are combined into a single statement fingerprint. [#120236][#120236]
+- An alert message is now displayed in the DB Console when a license is expired or there are fewer than 15 days left before it expires. [#120490][#120490]
+
+<h3 id="v24-1-0-alpha-4-bug-fixes">Bug fixes</h3>
+
+- Previously, [`DROP SCHEMA ... CASCADE`](../v24.1/drop-schema.html) could lead to dangling function references in other schemas accessing any functions. This is now fixed. [#119932][#119932]
+- Fixed an issue in which a [`RESTORE`](../v24.1/restore.html) of a backup that itself contained a table created by the `RESTORE` of a table with an in-progress [`IMPORT INTO`](../v24.1/import-into.html) would fail to restore all rows. [#120414][#120414]
```

Comment:
```suggestion
- Fixed a bug where a [`RESTORE`](../v24.1/restore.html) of a backup that itself contained a table created by the `RESTORE` of a table with an in-progress [`IMPORT INTO`](../v24.1/import-into.html) would fail to restore all rows. [#120414][#120414]
```